nifi-fn/pom.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.apache.nifi.fn</groupId>
+    <artifactId>nifi-fn</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
I think you would want to change this to three-component versioning as we follow [_semantic versioning_](https://semver.org/).

Reply:
Makes sense. Updated.